src/FuturesClient.ts

```diff
@@ -354,12 +354,21 @@ export class FuturesClient extends BaseRestClient {
     return this.postPrivate('api/v1/orders/multi', params);
   }
 
+  /**
+   * @deprecated, use cancelAllOrdersV3 instead
+   */
   cancelAllOrders(params?: {
     symbol?: string;
   }): Promise<APISuccessResponse<{ cancelledOrderIds: string[] }>> {
     return this.deletePrivate('api/v1/orders', params);
   }
 
+  cancelAllOrdersV3(params?: {
+    symbol?: string;
+  }): Promise<APISuccessResponse<{ cancelledOrderIds: string[] }>> {
+    return this.deletePrivate('api/v3/orders', params);
+  }
+
```

Comment:
If both are the same, input and output, I'd be tempted to actually change cancelAllOrders to use the v3 endpoint, and add a V1 function as backup (in case the new endpoint has issues)... thoughts?